### PR TITLE
Upgrade webpack & babel & eslint packages

### PR DESCRIPTION
**Pull request by donbot-inc**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tars",
-  "version": "1.9.11",
+  "version": "1.9.12",
```
Comment (artem-malko) [inline package.json]: Need to up to 1.10.0 To many fixes)

File: tars.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
     "name": "tars",
-    "version": "1.9.11",
+    "version": "1.9.12",
```
Comment (artem-malko) [inline tars.json]: Up to 1.10.0 too

**Comment by artem-malko** [comment]
You are awesome!!! The biggest pull request ever!) 
So, please, change version to 1.10.0 
Did you test it localy?

**Comment by donbot-inc** [comment]
> You are awesome!!! The biggest pull request ever!)
> So, please, change version to 1.10.0

Done in https://github.com/tars/tars/pull/367/commits/9c8b0e9c31a58b4592c4c8980a98b20dabb9ef2e

> Did you test it localy?

Yes. In my cases all work normally. I think it may need some work after the update, but I am ready to send a new pull request with bugfixes as soon as possible.

Also, the project structure does not allow easy testing of TARS on a local machine and I cannot be sure that update works 100%.

File: package.json (hunk 1 above)
Comment (donbot-inc) [reply package.json]: Done in 9c8b0e9

File: tars.json (hunk 2 above)
Comment (donbot-inc) [reply tars.json]: Done in 9c8b0e9

**Comment by artem-malko** [comment]
> Also, the project structure does not allow easy testing of TARS on a local machine and I cannot be sure that update works 100%.
Yes, I know( It was my first big open project) There were many ideas to make it much more simple in v2) But I need more time or developers for it)

**Comment by donbot-inc** [comment]
> Need to resolve conflicts. I think, it is a reason, why tests are failed)

Done

**Comment by donbot-inc** [comment]
I am waiting for approval of changes

**Comment by artem-malko** [comment]
So, everything is ok.
And now we have to wait, when all tests will be passed. We have to release both packages (tars-cli and tars) together.
So, remove NodeJS 9 from tars-cli CI.

After that, I will merge everything and publish new version.

**Comment by donbot-inc** [comment]
> So, remove NodeJS 9 from tars-cli CI.

Done.

**Comment by artem-malko** [comment]
So, let's start!)

**Comment by artem-malko** [comment]
Everything is published! You are awesome!)
By the way, do you have a telegram/email/facebook account or something like that?

**Comment by donbot-inc** [comment]
> By the way, do you have a telegram/email/facebook account or something like that?

Sure, https://t.me/donbot_inc

**Pull request merged**